modules/ROOT/pages/passing-additional-arguments-to-the-jvm-to-control-mule.adoc

```diff
@@ -0,0 +1,21 @@
+= Passing Additional Arguments to the JVM to Control Mule
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, deploy, configure, jvm arguments, system properties, properties
+
+//The Wrapper provides several properties you can set as described in the article on http://wrapper.tanukisoftware.org/doc/english/properties.html[Java Service Wrappers].
+
+If you want to pass additional arguments to the JVM that runs Mule, you can add `wrapper.java.additional.n` entries to the `wrapper.conf` file in the `${MULE_HOME}/conf` directory, or you can pass the arguments at the command line by adding the `-M` switch.
+
+For example, to set Mule's encoding:
+
+* Add `wrapper.java.additional.1=-Dmule.encoding=ISO-8859-1` to the Wrapper configuration file, or you could add `-M-Dmule.encoding=ISO-8859-1` to the Mule script at the command line.
+
+[CAUTION]
+If you add `wrapper.java.additional.n` entries to the configuration file, you must change each instance of `n` to a consecutive number, or Java does not parse the properties correctly.
+
+You can set Java Service Wrapper properties in the wrapper that you implement.
+
+== See Also
+* http://wrapper.tanukisoftware.org/doc/english/properties.html[Java Service Wrappers]
```

Comment:
We try to avoid links to 3rd party sites unless we really have to. Is there an alternative?